### PR TITLE
Create Roster and Player components

### PR DESCRIPTION
**Pull request by ritot**
- Added skeleton Roster component, which is made up of a grid of Player components
- Implemented smooth scrolling to Roster section on team pages
- Standardized player images to end in `.jpg`

*TODO: Add remaining players for WUFO and LaWUFA*

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -18,15 +18,14 @@ export default function Navbar(props) {
...
           <Link 
-            to='/lawufa'
+            smooth to={`/${props.team}#about`}
             className="bg-white p-2 shadow-drop text-gray-shadow transition hover:bg-purple-100 duration-125 ease-in-out"
           >
             About
           </Link>
           <Link 
-            to='/buf'
+            smooth to={`/${props.team}#roster`}
```
Comment (alansun25) [inline src/components/Navbar.jsx:28]: Clutch

File: src/pages/WUFO.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, {useRef} from 'react';
```
Comment (alansun25) [inline src/pages/WUFO.jsx:1]: Remove useRef

**Review by alansun25 — CHANGES_REQUESTED** [review]
Looks great, just gotta remove useRef from WUFO.jsx

File: src/pages/WUFO.jsx (hunk 2 above)
Comment (ritot) [reply src/pages/WUFO.jsx]: Done!

**Review by alansun25 — APPROVED** [review]
Esketit

**Pull request merged**